src/components/nodesColumns/columns.tsx

```diff
@@ -143,6 +201,50 @@ export function getCpuColumn<T extends {PoolStats?: TPoolStats[]}>(): Column<T>
         resizeMinWidth: 60,
     };
 }
+export function getTotalCpuColumn<T extends {PoolStats?: TPoolStats[]}>(): Column<T> {
+    return {
+        name: NODES_COLUMNS_IDS.TotalCPU,
+        header: NODES_COLUMNS_TITLES.TotalCPU,
+        sortAccessor: ({PoolStats = []}) => Math.max(...PoolStats.map(({Usage}) => Number(Usage))),
+        defaultOrder: DataTable.DESCENDING,
+        render: ({row}) => {
+            if (!row.PoolStats) {
+                return EMPTY_DATA_PLACEHOLDER;
+            }
+
+            const totalPoolUsage = row.PoolStats.reduce((acc, pool) => acc + (pool.Usage || 0), 0);
+
+            return (
+                <CellWithPopover
+                    placement={['top', 'auto']}
+                    fullWidth
+                    content={
+                        <DefinitionList responsive>
+                            {row.PoolStats.map((pool) =>
+                                isNumeric(pool.Usage) ? (
+                                    <DefinitionList.Item key={pool.Name} name={pool.Name}>
+                                        {formatPool('Usage', pool.Usage).value}
+                                    </DefinitionList.Item>
+                                ) : null,
+                            )}
+                        </DefinitionList>
+                    }
+                >
+                    <ProgressViewer
+                        value={totalPoolUsage}
+                        capacity={1}
+                        colorizeProgress
+                        percents
+                        vertical
+                    />
+                </CellWithPopover>
+            );
+        },
+        align: DataTable.LEFT,
+        width: 85,
+        resizeMinWidth: 40,
```

Comment:
It looks strange when resized. Let's do either with fixed bar width or non-resizable column.
<img width="589" alt="Screenshot 2024-10-31 at 18 16 58" src="https://github.com/user-attachments/assets/f0391712-79f3-4023-b11b-37a3471b52a1">

Reply:
reworked